sphinx/ext/apidoc.py

```diff
@@ -195,55 +210,55 @@ def create_modules_toc_file(
     else:
         template_path = [template_dir]
     text = ReSTRenderer(template_path).render('toc.rst_t', context)
-    write_file(name, text, opts)
+    return write_file(name, text, opts)
 
 
 def is_skipped_package(
-    dirname: str, opts: Any, excludes: Sequence[re.Pattern[str]] = ()
+    dirname: str | Path, opts: CliOptions, excludes: Sequence[re.Pattern[str]] = ()
 ) -> bool:
     """Check if we want to skip this module."""
-    if not path.isdir(dirname):
+    if not Path(dirname).is_dir():
         return False
 
-    files = glob.glob(path.join(dirname, '*.py'))
+    files = glob.glob(str(Path(dirname, '*.py')))
```

Comment:
It's possible to directly use something like `Path(dirname).glob('*.py')`.

Reply:
The thing I was worried about here, is that I believe they have slightly different logic, e.g.  `glob.glob` does not include hidden files by default:

- https://docs.python.org/3/library/glob.html#glob.glob
- https://docs.python.org/3/library/pathlib.html

Reply:
Mmmh. Ok, le'ts keep what has been working for years!